pkg/activator/throttler.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package activator
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	v1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/knative/serving/pkg/queue"
+	"github.com/knative/serving/pkg/reconciler"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	capacityUpdateFailure = "updating capacity failed"
+	// OverloadMessage indicates that throttler has no free slots to buffer the request
+	OverloadMessage = "activator overload"
```

Comment:
Can this be a sentinel error and reused below?

Reply:
Every occurrence of `errors.New(OverloadMessage)`

Reply:
I think it is only used in one place.

Reply:
then why bother making it a constant? 🤔

Reply:
It should be used in more then one place in the next PR, I didn't update it for this one. I'd propose to leave it as const for now.